lib/plugins/init/tests/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Init = require('../index.js');
+const Serverless = require('../../../Serverless');
+
+describe('Init', () => {
+  let init;
+  let serverless;
+
+  beforeEach(() => {
+    serverless = new Serverless();
+    init = new Init(serverless);
+  });
+
+  describe('#constructor()', () => {
+    it('should have commands', () => expect(init.commands).to.be.not.empty);
+  });
+
+  describe('#init()', () => {
```

Comment:
Sure, but it has to be fixed before we can merge it. Can't be merged without proper tests in place.